frontend/src/__tests__/cypress/cypress/utils/dataLoader.ts

```diff
@@ -1,11 +1,18 @@
 import yaml from 'js-yaml';
-import type { DataScienceProjectData } from '~/__tests__/cypress/cypress/types';
+import type { DataScienceProjectData, ResourcesData } from '~/__tests__/cypress/cypress/types';
 
-// Load fixture function that returns a specific type
+// Load fixture function that returns DataScienceProjectData
 export const loadFixture = (fixturePath: string): Cypress.Chainable<DataScienceProjectData> => {
```

Comment:
It's better to use a more specific name: loadDSPFixture or something like taht

Reply:
Good catch, I've updated this along with the other places it was referenced.